controllers/managedocs_controller.go

```diff
@@ -1278,26 +1280,27 @@ func (r *ManagedOCSReconciler) findOCSVolumeClaims() (bool, error) {
 	if err := r.UnrestrictedClient.List(r.ctx, &storageClassList); err != nil {
 		return false, fmt.Errorf("unable to list storage classes: %v", err)
 	}
-	// create slice of ocs storage classes by checking provisioner
+
+	// create map of ocs storage classes by checking provisioner
 	// Provisioner will be prefixed by namespace
-	ocsStorageClass := []string{}
+	ocsStorageClass := make(map[string]bool)
 	for i := range storageClassList.Items {
 		if strings.HasPrefix(storageClassList.Items[i].Provisioner, r.namespace) {
-			ocsStorageClass = append(ocsStorageClass, storageClassList.Items[i].Name)
+			ocsStorageClass[storageClassList.Items[i].Name] = true
 		}
```

Comment:
I think the above suggestion is more readable

```suggestion 
		storageClass := &storageClassList.Items[i]
		if strings.HasPrefix(storageClass.Provisioner, r.namespace) {
			ocsStorageClass[storageClass.Name] = true
		}
```